packages/semantic-tokens/scripts/tokenGen.ts

```diff
@@ -113,8 +127,8 @@ const tokenExport = (token: string, resolvedTokenFallback: string) => {
 };
 
 const getResolvedToken = (token: string, tokenData: Token, tokenNameRaw: string) => {
-  const tokenSemanticRef =
-    tokenData.fst_reference.length > 0 ? toCamelCase(cleanFstTokenName(tokenData.fst_reference)) + 'Raw' : null;
+  const isValidToken = !tokenData.fst_reference.startsWith('NULL/') && tokenData.fst_reference.length > 0;
```

Comment:
Even though this is somewhat the inverse of `isInvalidToken`, it feels like we can combine those a bit and have a comprehensive test?